docs/source/topics/ref_dns-configuration.adoc

```diff
@@ -17,7 +17,29 @@ Additional checks are done to verify DNS is properly configured when running [co
 [id="dns-configuration-linux_{context}"]
 == Linux
 
-On Linux, {prod} expects the following DNS configuration:
+On Linux, depending on your distribution, {prod} expects the following DNS configuration.
+
+=== NetworkManager + systemd-resolved
+
+This configuration is used on Fedora 33 or newer, and on Ubuntu Desktop editions.
+
+* {prod} expects NetworkManager to manage networking.
+* {prod} configures systemd-resolved  to forward requests for the `testing` domain to the `192.168.130.11` DNS server. `192.168.130.11` is the IP of the {prod} virtual machine.
+* systemd-resolved configuration is done through a NetworkManager dispatcher script in `/etc/NetworkManager/dispatcher.d/pre-up.d/99-crc.sh` so that it persists over reboots:
++
+----
+if [ "$1" = crc ]; then
+        resolvectl domain "$1" ~testing
+        resolvectl dns "$1" 192.168.130.11
+        resolvectl default-route "$1" false
+fi
+
+exit 0
+----
+
+=== NetworkManager + dnsmasq
+
+This configuration is used on Fedora 32 or older, on {rhel} and on {centos}.
```

Comment:
Missing a comma: "`This configuration is used on Fedora 32 or older, on {rhel}, and on {centos}.`"